testing/sdk_jobs.py

```diff
@@ -71,14 +72,13 @@ def __exit__(self, *args):
 def run_job(job_dict, timeout_seconds=600, raise_on_failure=True):
     job_name = job_dict['id']
 
-    sdk_cmd.run_cli('job run {}'.format(job_name))
-
-    def wait_for_run_id():
-        runs = json.loads(sdk_cmd.run_cli('job show runs {} --json'.format(job_name)))
-        if len(runs) > 0:
-            return runs[0]['id']
-        return ''
-    run_id = shakedown.wait_for(wait_for_run_id, noisy=True, timeout_seconds=timeout_seconds, ignore_exceptions=False)
+    run_id_pattern = re.match('^Run ID: (.*)$', sdk_cmd.run_cli('job run {}'.format(job_name)))
```

Comment:
starting the job with --json seems less evil.  shakedown already has code to do this.
Just my 2 cents.

Reply:
Oh cool. Apparently --json was added sometime in 0.5. I'll try that out (to get some sample output) and then update this PR to use it.